bot.py

```diff
@@ -204,6 +211,7 @@ async def handle_message(self, message):
             return await message.channel.send(f":robot: {error_msg}")
 
         # Execute the command.
+        self.last_command_time = time.time()
```

Comment:
Should the latest command time be registered if the command is not voice related? Should `!joke` reset the command timer?

Reply:
I think it should, yes. I would find it weird if the bot would randomly disconnect while it was doing somethinge else, even if it wasn't voice related.

Reply:
Also, I want to try and minimize how often the users have to hear the disconnect chime in discord, while still keeping the bot feeling responzive.